M2/Macaulay2/packages/undistributed-packages/ParameterSchemes.m2

```diff
@@ -243,11 +243,11 @@ parameterIdeal(Ideal,Ideal) := (M,family) -> (
      -- M is a monomial ideal in a polynomial ring
      -- family is the result of a call to 'parameterFamily'
      R := ring M;
-     time G = forceGB gens family;
-     time syzM = substitute(syz gens M, ring family);
-     time eq = compress((gens family * syzM) % G);
+     time G := forceGB gens family;
+     time syzM := substitute(syz gens M, ring family);
```

Comment:
This package is perhaps superseded by GroebnerStrata.m2, but in fact there might be some things here that I want to include there.  I had forgotten that this package was here.